src/niscope/metadata/functions_addon.py

```diff
@@ -62,6 +62,7 @@
     'FetchBinary32':                    { 'codegen_method': 'no',       },  # TODO(marcoskirsch):No support for fetching binary. Issue #511
     'ActualMeasWfmSize':                { 'codegen_method': 'private',  },  # We use it internally so the customer doesn't have to.
     'ActualNumWfms':                    { 'codegen_method': 'private',  },  # We use it internally so the customer doesn't have to.
+    '.etAttributeViInt64':              { 'codegen_method': 'no',       },  # NI-Scope has no ViInt64 attributes.	
```

Comment:
Product is called NI-SCOPE. It's not your fault, there's no consistency in the spelling of the names of these drivers.